### DISCUSSION TITLE
Questions about conflict analysis

### DISCUSSION THREAD

**Discussion by tusiqi1** [ERGO-Code/HiGHS#1361] [Q&A]
Why does the set covering constraint generated for conflict analysis only perform domain propagation, but not the cut plane?

Is it because, as the result of paper “Confilct Analysis in Mixed Integer Programming” shows, although the number of node passes will be reduced, the computation time will be longer?

**Comment by tusiqi1** [comment]
Are there any relevant papers?

**Comment by jajhall** [comment]
I've no idea

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
